src/componets/list.js

```diff
@@ -24,85 +24,77 @@ export default class List extends React.Component {
     this.setState({ password: evt.target.value });
   }
 
-   handleSubmit = (evt) => {
-    if (!this.canBeSubmitted()) {
+  handleSubmit = (evt) => {
+    if (this.canBeSubmitted()) {
```

Comment:
earlier you had `(!this.canBeSubmitted())` that negated the boolean (so you'd never had had the chance to submit with that there.